src/components/list/List.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const List = ({title, children, classname}  ) => {
+    return (
+        <div className={classname}>
+            <h3> {title} </h3>
+            {children}
```

Comment:
Wat dacht je ervan om deze nóg efficiënter te maken? 😍  Dat kun je doen door de List zo op te stellen dat hij een array van objecten verwacht. Ieder object bevat de volgende properties: `url`, `name`, `iconOnly` (want pagina's binnen de applicatie wil je niet in een nieuw venster openen). Dan zorg je dat deze List over die array heen mapt en alle `<ResourceLink>`'s genereerd!